bin/validate_fom_components_yaml.R

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env Rscript
```

Comment:
Not a new file, created from https://github.com/pinin4fjords/shinyngs/blob/develop/exec/validate_fom_components.R